CardGameGUI.java

```diff
@@ -309,10 +309,8 @@ public void actionPerformed(ActionEvent e) {
 			getRootPane().setDefaultButton(replaceButton);
 			winMsg.setVisible(false);
 			lossMsg.setVisible(false);
-			if (!board.anotherPlayIsPossible()) {
-				signalLoss();

```

Comment:
Should we still signal a loss, that it counts as a loss?  Maybe leave a 5 second message/animation, then automatically redeal?  This change will affect the simulation statistics for how often you can win the game.